README.md

```diff
@@ -193,6 +193,12 @@ If you'd like to work with the previous implementation see the [Docker documenta
 
 Run `make lint`
 
+### Running jobs
+
+We run background jobs / workers with ActiveJob and GoodJob: `bundle exec good_job start`
```

Comment:
maybe comment that locally this is in the `Procfile` too?

Reply:
Meaning that `good_job` does not have to be started manually locally? Because the `Procfile` is executed in the local dev environment?

Reply:
yes both of those and also "here's an example in this repo of us using that"

Reply:
and to further connect, since we typically develop via `make run` which runs the Procfile, that's why we don't need to separately run this when developing